.husky/pre-push

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env sh
-
+pnpm check-documentation-map &&
 pnpm check-lock-files &&
 pnpm check-commit &&
-pnpm documentation &&
 pnpm pretty-quick --check
```

Comment:
Added check instead of generate to remove the revert issue

Reply:
Please revert this. It must be something specific about your setup/environment, other users do not reorder properties when they run this. I just verified on my machine

Reply:
Can't or else it generates files.
Please expect this as a check instead a write command on the git hook.

Reply:
That's what I'm saying though, it doesn't for anyone else (I pulled down this branch and ran it myself), each PR that goes into this repo is running the same commands. There is something unique about it running on your machine. We can figure that our separately, but for this PR we won't be making this change

Reply:
You can push with `--no-verify` after you make the change to make sure your unique issue doesn't impact the files

Reply:
`git push --no-verify`

Reply:
No, it's manditory for the CI as well.
I tried...

It's green with only 11 files now

Reply:
Gotcha. Sorry misunderstood. I will remove this change.